### DISCUSSION TITLE
具体来说，制作目标是什么？

### DISCUSSION THREAD

**Discussion by MY1L** [Hulenkius/ngiugyat#2] [General]
如这个问题：https://github.com/Hulenkius/ngiugyat/issues/1
我其实是觉得「无」不太美观……
如果你的目标是还原碑文，那可以用异体选择符指个异体；
如果你的目标是实用字体，那就把原碑文作为异体。

**Comment by Hulenkius** [comment]
思考……
魏碑真是很棘手的字體。字形太規整則失其拙趣，字形太散漫則粗劣不堪。
此項目的基本部件庫目前還不完善，計劃在描摹大量碑刻後做一次筆畫和間架的規整（不可下手太狠），這樣就可以在保留魏碑風格的情況下實用化。
目前的魏碑電腦字庫多是汲取後世書家的「魏體」風格的毛筆字。而其他碑體魏楷（華康龍門石碑）則太規整而且中宮臃腫不堪。
吾寧允執厥中。

嗯，附加一句，標上了「改製」標籤的字，基本上是經過了優化的。做成這個樣子就可以了。
